chaosaws/cloudwatch/probes.py

```diff
@@ -46,6 +46,18 @@ def get_metric_statistics(namespace: str, metric_name: str,
     Example: A duration of 60 seconds and an offset of 30 seconds will yield a
     statistical value based on the time interval between 30 and 90 seconds in the past.
 
+        *namespace: The AWS metric namespace.
+        *metric_name: The name of the metric to pull data for.
+        *dimensions: Are expected as:
+            Name: The name of the dimension to search for.
+            Value: The value to be used for searching the dimension.
+        *unit: The type of unit desired to be collected.
+        *statistic: The type of data to return.
+            One of: Average, Sum, Minimum, Maximum, SampleCount.
+        *period: The window in which to pull datapoints for.
+        *offset: The time (seconds) to offset the endtime (from now).
+        *duration: The time (seconds) to set the start time (from now).
```

Comment:
Could you please put the parameters in this style: https://github.com/chaostoolkit-incubator/chaostoolkit-aws/blob/master/chaosaws/awslambda/actions.py#L152-L158

Reply:
I only ask this, because the `*name: explanation` format is new to me and I've seen the linked style not only in the `chaostoolkit-aws` codebase, but others also 😄